automator/automator.py

```diff
@@ -22,7 +22,7 @@ class Automator(object):
       
     The recording is more directly controlled by the coordinator.
     The automator instructs the coordinator when it can record by setting
-    an `nshot` key in redis.
+    an `enabled` key in redis.
```

Comment:
you mean a "rec_enabled" key? at first I was confused if "enabled" and "rec_enabled" were two different things

Reply:
Yes, you're right, I meant `rec_enabled`